mock/py/mockbuild/podman.py

```diff
@@ -51,32 +55,46 @@ class Podman:
     def __init__(self, buildroot, image):
         self.podman_binary = "/usr/bin/podman"
         if not os.path.exists(self.podman_binary):
-            raise BootstrapError(f"'{self.podman_binary}' not installed")
+            raise PodmanError(f"'{self.podman_binary}' not installed")
 
         self.buildroot = buildroot
         self.image = image
-        self.container_id = None
-        getLog().info("Using bootstrap image: %s", image)
+        self.image_id = None
+        getLog().info("Using container image: %s", image)
 
     @traceLog()
     def pull_image(self):
         """ pull the latest image, return True if successful """
         logger = getLog()
         logger.info("Pulling image: %s", self.image)
         cmd = [self.podman_binary, "pull", self.image]
-        out, exit_status = util.do_with_status(cmd, env=self.buildroot.env,
-                                               raiseExc=False, returnOutput=1)
-        if exit_status:
-            logger.error(out)
-        return not exit_status
 
-    def import_tarball(self, tarball):
+        res = subprocess.run(cmd, env=self.buildroot.env,
+                             stdout=subprocess.PIPE, stderr=subprocess.PIPE,
```

Comment:
Hmm, why you do not use `util.do()` here?

Reply:
We started using `subprocess` in `podman.py` long time ago, and I think it was because
we wanted to make it easy and self-standing python module without the rest of `mockbuild`.
At least it complicated unit testing, from what I remember.

The util.do() does just too many opinionated things which are not needed here.